src/site/content/en/handbook/web-dev-components/index.md

```diff
@@ -37,6 +37,16 @@ guidance about how to use them effectively.
 1. [Tooltips](#tooltips)
 1. [Video](#video)
 
+## Code Patterns
```

Comment:
This needs to be added to the table of contents above. The headings are alphabetized so this item should come before the Code Pen shortcode in the doc